### ISSUE TITLE
Layout & Resize: Percentage based sizes instead absolute ones

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#4758]
TL;DR: 

As a VSCode user, I would like the editor to save the "sizes" of the panels to values relative to the screen being used, so that when you move the window between different monitors these proportions are respected.

**Related**: https://github-com/microsoft/vscode/-/67365, https://github-com/microsoft/vscode/-/14452

**Keywords**: Percentage, Screen Size, Panels, Relative size.

Full Story:

I'm using VSCode in a multi-monitor setup, as like as many others. [In particular I have a main widescreen monitor, bellow is my Macbook and also I have a portrait secondary monitor on the right side](https://www.reddit.com/r/macbookpro/comments/w7ghhl/displaylinkm1_my_triple_monitor_setup_with_the/).

Depending on the task I'm doing I need to move VSCode window between monitors.

I've appreciated that panel sizes are stored in absolute pixels or something similar, so _every time_ I need to move VSCode screen I do need to resize both lower and side panels. The most annoying one is when I move portrait VSCode to the MBP screen (see screenshots bellow), when Terminal panel gets the whole height instead "50%", as per portrait screen configuration. 

![Captura de Pantalla 2022-07-26 a las 10 20 09](https://user-images.githubusercontent.com/4610108/180960146-c4e3cb1f-a013-4a29-a9ef-1b055964096c.png)
<img width="1552" alt="Captura de Pantalla 2022-07-26 a las 10 25 58" src="https://user-images.githubusercontent.com/4610108/180960153-7c7f231b-6c89-4c79-9ae9-20d6978e464f.png">
